metadata.json

```diff
@@ -22,7 +22,7 @@
     },
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.0 < 9.0.0"
+      "version_requirement": ">= 4.13.0 < 10.0.0"
```

Comment:
Had to allow 9.x cause without it `puppet module install` in acceptance never finishes ...